composer.json

```diff
@@ -68,7 +68,7 @@
         "webmozart/assert": "^1.3"
     },
     "conflict": {
-        "phpunit/php-code-coverage": ">9,<9.1.4 || >9.2.17",
+        "phpunit/php-code-coverage": ">9,<9.1.4 || >9.2.17,<9.2.20",
```

Comment:
I may have missed something, but I think sebastianbergmann/php-code-coverage#964 is not released yet, and will probably be in **9.2.21**, so I'd suggest `<9.2.21` or `<=9.2.20`.

Reply:
AHA! Thanks: missed that :) 

Yes, will adjust after release.

Reply:
Force-pushed :)